src/pybind/mgr/dashboard_v2/controllers/cephfs.py

```diff
@@ -0,0 +1,322 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import collections
+from collections import defaultdict
+import json
+import time
+
+import cherrypy
+from mgr_module import CommandResult
+
+from ..tools import ApiController, AuthRequired, BaseController, NotificationQueue, ViewCache
```

Comment:
Remove ``NotificationQueue``, it's not needed

Reply:
Done.